src/Uno.UI/Microsoft/UI/Xaml/Controls/RatingControl/RatingControlAutomationPeer.cs

```diff
@@ -197,7 +198,7 @@ private string GenerateValue_ValueString(string resourceString, double ratingVal
 		SignificantDigitsNumberRounder rounder = new SignificantDigitsNumberRounder();
 		formatter.NumberRounder = rounder;
 
-		string maxRatingString = GetRatingControl().MaxRating.ToString();
+		string maxRatingString = GetRatingControl().MaxRating.ToString(CultureInfo.InvariantCulture);
```

Comment:
This particular one is likely to presented to the user. It will need to be `CurrentUICulture`.

Reply:
```suggestion
		string maxRatingString = GetRatingControl().MaxRating.ToString(CultureInfo.CurrentUICulture);
```